Kudu.Core/SourceControl/Git/LibGit2SharpRepository.cs

```diff
@@ -239,9 +240,15 @@ public void FetchWithoutConflict(string remoteUrl, string branchName)
             }
             catch (LibGit2SharpException exception)
             {
-                if (exception.Message.Equals("Unsupported URL protocol"))
+                tracer.TraceWarning("LibGit2SharpRepository fetch failed with {0}", exception);
+
+                // LibGit2Sharp doesn't support SSH yet. Use GitExeRepository
+                // LibGit2Sharp only supports smart Http protocol
+                if (exception.Message.Equals("Unsupported URL protocol") ||
```

Comment:
instead of handle case by case. shall we always fallback to use git.exe when fetch fail?